src/cljs/main/broadfcui/page/workspace/summary/attribute_editor.cljs

```diff
@@ -102,11 +102,15 @@
              with-spaces {:error (str "Keys cannot have spaces: " (string/join ", " with-spaces))}
              invalid-numbers {:error (str "Invalid number for key(s): " (string/join ", " invalid-numbers))}
              :else {:success typed})))
+   :get-initial-state
+   (fn []
+     {:table-key (gensym)})
```

Comment:
I dunno if you need this in initial state, since you set it for any data changes ... but I *like* it here, for readability and understandability.